tests/cryptolibAESSHA/mmcau_aes_functions.S

```diff
@@ -0,0 +1,929 @@
+#*******************************************************************************
+#*******************************************************************************
+#
+# Copyright (c) Freescale Semiconductor, Inc 2011.
```

Comment:
under what license does Freescale distribute this assembler file?

Reply:
Posted the license. It's permissive.